locales/en/public.json

```diff
@@ -236,20 +236,20 @@
     "RESOLVE_MESSAGE": "Reload the page and try again. If the error still persists, see the list of <issue>{{knownIssue}}</issue> or <report>{{fileReport}}</report>."
   },
   "ErrorView": {
-    "EVENT_TEMPLATES": "Error retrieving event templates"
+    "EVENT_TEMPLATES": "Error retrieving Event Templates"
   },
   "JvmDetailsCard": {
     "CARD_DESCRIPTION": "Display details about the selected target JVM.",
-    "CARD_DESCRIPTION_FULL": "View information such as the connection URL, labels, and annotations belonging to the selected target JVM.",
-    "CARD_TITLE": "Target JVM Details"
+    "CARD_DESCRIPTION_FULL": "View information such as the connection URL, Labels, and annotations belonging to the selected target JVM.",
```

Comment:
It seems like labels and annotations are sort of equivalent - if one deserves capitalization then the other should, too?

Reply:
Yes, I think the best thing is I should revisit [https://github.com/cryostatio/cryostatio.github.io/pull/153](https://github.com/cryostatio/cryostatio.github.io/pull/153) and related PRs and see what we did on these.